spec/lib/tasks/user_spec.rb

```diff
@@ -47,4 +48,32 @@
       @rake[task_name].invoke
     end
   end
+
+  describe 'usasearch:user:update_inactive_approval_status' do
+    let(:task_name) { 'usasearch:user:update_inactive_approval_status' }
+
+    before do
+      @rake[task_name].reenable
+      User.destroy_all("email != 'inactivedeveloper@fixtures.org'")
+    end
+
+    it "has 'environment' as a prereq" do
+      expect(@rake[task_name].prerequisites).to include('environment')
+    end
+
+    it 'sets inactive users to not_approved' do
+      @rake[task_name].invoke
```

Comment:
@peggles2 , this change is at your discretion, but this would be a good time to use the `change` method for the expectation:
```ruby
      expect{ @rake[task_name].invoke }.
        to change { inactive_user.is_not_approved? }.from(false).to(true)
```
I like that format because it ensures that the expected pre-condition is correct. For example, if someone changed the relevant fixture to have `approval_status: not_approved`, the original spec could pass as a false positive. The version using `change` would not.